#Yoyo parameters
axle_radius: 0.0055                     #axle radius of the yoyo
string_length: 1.0                      #string length of the yoyo
robot_origin_from_ground: 1.0           #robot origin from the ground
